charts/stable/monica/values.yaml

```diff
@@ -9,7 +9,7 @@ image:
   # -- image repository
   repository: monica
   # -- image tag
-  tag: 3.1.1-apache
+  tag: 3.7.0-apache
```

Comment:
I would like to suggest changing this to the following (GitHub wouldn't let me create an actual suggestion for some reason), that way we only have to maintain the appVersion in the `Chart.yaml` file:

```yaml
  # -- image tag
  # @default -- chart.appVersion
  tag:
```